src/javascript/app_2/Stores/Modules/SmartChart/smart-chart-store.js

```diff
@@ -92,6 +98,30 @@ export default class SmartChartStore extends BaseStore {
         this.barriers = {};
     }
 
+    @action.bound
+    saveAndClearTradeChartLayout() {
+        this.should_save_layout   = true;
+        this.should_import_layout = false;
+        this.trade_chart_layout   = {};
+        this.trade_chart_symbol   = this.root_store.modules.trade.symbol;
+    }
+
+    @action.bound
+    applySavedTradeChartLayout() {
+        this.should_import_layout = true;
+        this.should_clear_chart   = false;
+        this.should_save_layout   = false;
```

Comment:
can we follow the same order of assignment in `saveAndClearTradeChartLayout` so we know what is different upon line-by-line check?